src/VIPServer.java

```diff
@@ -107,24 +107,6 @@ public void main(String[] args) throws HostFailureException, HostNotFoundExcepti
 					// SHOULD BE REPLACED BY
 					// job.carryOn();
 				} else {
-					if (!timer) {
```

Comment:
What is in GATE_DISCONNECT assumed that the Merge process was launched at the beginning of the simulation and waits to be activated. If the process is launched at its start_time, they are 2 options:
1) it is before the last GATE_DISCONNECT and the current code handles this case
2) it is after the last GATE_DISCONNECT and then we have to test if gate jobs are done and if yes, start the process:
if (endedGateWorkers == VIPSimulator.numberOfGateJobs && )